lte/gateway/python/Makefile

```diff
@@ -11,7 +11,10 @@ define setup_rules
 endef
 _ := $(foreach python_src, $(PYTHON_SRCS), $(eval $(call setup_rules,$(python_src))))
 
-buildenv: setupenv protos swagger $(BUILD_LIST) py_patches
+snowflake:
+	. ${PYTHON_BUILD}/bin/activate && sudo env PATH=${PATH} snowflake --make-snowflake
+
+buildenv: setupenv protos swagger $(BUILD_LIST) py_patches snowflake
```

Comment:
This is run during ansible provisioning - is it?

Reply:
I thought it would be run during the manually triggered `make run` step after ansible provisioning.

Just searched for `make {all,build,run,build_python,buildenv}` in lte/gateway/deploy and orc8r/tools/ansible/roles and got no hits that suggested it is run.

But I might be missing something?